src/app/tab-preferences/notification-base.ts

```diff
@@ -195,6 +198,48 @@ export class NotificationBase implements OnInit {
     }
   }
 
+  // TODO: This should work with a <string, boolean> map but that caused problems with the ionic toggles
```

Comment:
This is SUCH a pain in the a.. If you think that this should be easy, feel free to change that.